backend-project/small_eod/institutions/tests/test_serializers.py

```diff
@@ -0,0 +1,174 @@
+from django.test import TestCase
+from rest_framework.test import APIRequestFactory, force_authenticate
+
+from ..models import AddressData, ExternalIdentifier, Institution
+from ..serializers import (
+    InstitutionSerializer,
+    ExternalIdentifierNestedSerializer,
+    AddressDataNestedSerializer,
+)
+from ...users.factories import UserFactory
+from teryt_tree.factories import JednostkaAdministracyjnaFactory, CategoryFactory
+
+
+class AddressDataSerializerTestCase(TestCase):
+    def setUp(self):
+        self.default_serializer = AddressDataNestedSerializer(
+            data={
+                "email": "asd@asd.pl",
+                "city": "City1",
+                "epuap": "epuap",
+                "street": "street",
+                "house_no": "11",
+                "postal_code": "22222",
+                "flat_no": "",
+            }
+        )
+        self.obj = (
+            self.default_serializer.save()
```

Comment:
Mam wrażenie, że wykonywanie w tym miejscu metody ```save``` z serializera to nie jest najlepszy pomysł, bo istnieje duża szansa, że ta metoda będzie zepsuta, a wtedy cała klasa będzie zgłaszać błąd, a wtedy testy nie będą pomagać w ustaleniu skali błędu.